packages/valory/skills/abstract_round_abci/tests/test_behaviours_utils.py

```diff
@@ -338,8 +338,8 @@ def async_act(self) -> Generator:
     assert behaviour.counter == 1
     assert behaviour.state == AsyncBehaviour.AsyncState.RUNNING
 
-    # wait the sleep timeout
-    time.sleep(timedelta)
+    # wait the sleep timeout, we give twice the amount of time it takes the behaviour
+    time.sleep(timedelta * 2)
```

Comment:
The problem was that sometimes this sleep finishes before the sleep above (line 325).